docs/config-file-copyright-garbage-yml.md

```diff
@@ -6,7 +6,7 @@ You can use the [copyright-garbage.yml example](../examples/copyright-garbage.ym
 
 ### When to Use
 
-No scanner is perfect and many detect code statements or binary code as copyright holders,
+No scanner is perfect, and many detect code statements or binary code as copyright holders,
```

Comment:
Actually, I believe we must not use a comma after "and" here as the second clause is not independent from the first one, see the (incorrect) "Sam tossed the ball, and watched the dog chase it." example [here](https://www.grammarly.com/blog/comma-before-and/).

Reply:
Right. Done.